Cargo.toml

```diff
@@ -25,7 +25,7 @@ path = "src/bin.rs"
 bitcoin = "0.30"
 
 # Optional dependencies
-serde = { version = "1", optional = true }
+serde = { version = "1", optional = true, features = ["derive"] }
```

Comment:
where is derive feature used?

Reply:
When using "cargo install" I always got errors about Serialization macros not being available. 
https://github.com/weareseba/livedeb/blob/0a869312b0e5de5ca5c195109e5e88d7829a00f6/Dockerfile#L38
I could only fix those errors by adding this derive feature.

Reply:
Oh indeed Serialize/Deserialize are used in the electrum wallet file handling